server/services/category.js

```diff
@@ -0,0 +1,14 @@
+const Model = require('../models/category');
+
+module.exports = {
```

Comment:
Made a comment about these service methods in our group slack but I will also put it here:
We don't want to throw errors like this in the service layer. We also don't want to use express at all in the service layer. Instead we want to define our service errors like in this [example](https://github.com/TritonSE/LAK-Goods-Transport-Application/blob/main/backend/errors.js) . 

This readme has some more information: https://github.com/TritonSE/LAK-Goods-Transport-Application/tree/main/backend#readme on error handling. After defining these errors, you can throw them from the service layer. You also don't need module.exports here, you can just think of the service layer like a collection of async functions that call Mongo functions. 

You can then catch and deal with them in your route like this: https://github.com/TritonSE/LAK-Goods-Transport-Application/blob/main/backend/routes/job.js